### DISCUSSION TITLE
Is there Record.values in fp-ts?

### DISCUSSION THREAD

**Discussion by adinvadim** [gcanti/fp-ts#1572] [Q&A]
I found Record.keys, but i cant find Record.values, maybe function has a different name?

**Comment by steida** [comment]
Check source code and tests.

**Comment by samhh** [comment]
I don't _think_ it exists yet. For the timebeing it's present in [fp-ts-std](https://samhh.github.io/fp-ts-std/modules/Record.ts.html#values).

**Comment by edalex-ian** [comment]
May be possible to show you an alternate approach if you provide context on how you want to use the output from `Record.values`.

But otherwise, if you didn't want to bring in `fp-ts-std` you could use [collect](https://gcanti.github.io/fp-ts/modules/Record.ts.html#collect).
